cypress/support/pages/admin_console/manage/users/UserDetailsPage.ts

```diff
@@ -0,0 +1,54 @@
+import { RequiredActionAlias } from "keycloak-admin/lib/defs/requiredActionProviderRepresentation";
+
+export default class UserDetailsPage {
+    saveBtn: string;
+    cancelBtn: string;
+    emailInput: string;
+    emailValue: string;
+    firstNameInput: string;
+    firstNameValue: string;
+    lastNameInput: string;
+    lastNameValue: string;
+    enabledSwitch: string;
+    enabledValue: boolean
+    requiredUserActions: RequiredActionAlias[];
+  
+    constructor() {
+      this.saveBtn = "[data-testid=save-user]";
```

Comment:
I've introduced an extension method for this you can set this to:
```suggestion
      this.saveBtn = "save-user";
```